spring-cloud-starter-tencent-polaris-router/src/main/java/com/tencent/cloud/polaris/router/feign/RouterLabelFeignInterceptor.java

```diff
@@ -102,22 +105,20 @@ public void apply(RequestTemplate requestTemplate) {
 				.getFragmentContext(MetadataContext.FRAGMENT_TRANSITIVE);
 		labels.putAll(transitiveLabels);
 
-		// Because when the label is placed in RequestTemplate.header,
-		// RequestTemplate will parse the header according to the regular, which conflicts with the expression.
-		// Avoid conflicts by escaping.
-		Map<String, String> escapeLabels = new HashMap<>(labels.size());
-		for (Map.Entry<String, String> entry : labels.entrySet()) {
```

Comment:
delete `com.tencent.cloud.common.util.ExpressionLabelUtils#escape` method

Reply:
In 2020.0 or Greenwich branch? Can it be reviewed in main branch, if it's ok, merge it into 2020.0 or Greenwich?